examples/blog/blog.js

```diff
@@ -19,6 +21,17 @@ async function get (req, res) {
     if (from || to) {
       console.log(`filter: ${from && from.value} - ${to && to.value}`)
     }
+
+    req.hydra.resource.dataset.add(rdf.quad(
+      req.hydra.term,
+      vocab.hydra.view,
+      url,
+    ))
+    req.hydra.resource.dataset.add(rdf.quad(
+      url,
+      vocab.rdf.type,
+      vocab.hydra.PartialCollectionView,
+    ))
```

Comment:
```turtle
<> hydra:view </?from=foo;to=bar> .
</?from=foo;to=bar> a hydra:PartialCollectionView .
```

A client requests `/?from=foo;to=bar`
This is the way to tell the client that it should really be interested in the `<>` resource and not the requested on (with query).

The `rdf:type` triple should be unnecessary but needs a change in Alcaeus